build.gradle

```diff
@@ -46,6 +46,22 @@ dependencies {
     implementation "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
     implementation "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
     implementation "javax.inject:javax.inject:1"
+    implementation("org.apache.tika:tika-parsers:1.22") {
```

Comment:
@ahoffer Starting a thread to respond to https://github.com/connexta/ion-search/pull/86#issuecomment-558728222:

We've been ignoring snyk findings so far across all of our repos. They are not required checks for PRs.